docs/html/_sources/pages/install.rst.txt

```diff
@@ -0,0 +1,97 @@
+sncli
```

Comment:
this file seems to be duplicating `docs/html/_sources/install.rst.txt`.

side note: should the files have a `.rst.txt` extension, or `.rst`?

Reply:
The 'html` folder was orginally called `build`, I changed it so `git` would add it. Every file in that folder was created by the `sphinx-build` command. I'm not sure what can all be deleted, but if the end goal is to upload this to ReadtheDocs, maybe only the `source` files will be needed?

Reply:
Yes, we only want source files in the repo. Any build artefacts should be git-ignored.